webui/react/src/components/JupyterLabModal.tsx

```diff
@@ -171,7 +174,7 @@ const JupyterLabModalComponent: React.FC<Props> = ({ workspace }: Props) => {
     if (showFullConfig) {
       fetchConfig();
     }
-  }, [fetchConfig, showFullConfig]);
+  }, [fetchConfig, showFullConfig, currentWorkspace]);
```

Comment:
I'm a little worried because this updates from `currentWorkspace` but `fetchConfig` reads from `workspaceId: fields.workspaceId,`
Is there some way to pass `currentWorkspace` or change based on the same variable?